packages/@ama-sdk/swagger-builder/src/core/swagger-spec-merger.ts

```diff
@@ -130,7 +130,6 @@ export class SwaggerSpecMerger {
           break;
         }
 
-        case 'paths':
```

Comment:
I think we should, in contrary, have a rule to enforce expliciting all the possible cases of a switch case

Reply:
[unicorn/no-useless-switch-case](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v55.0.0/docs/rules/no-useless-switch-case.md)

Reply:
I can see value for both:
- having it explicitely written ensure that the developer did not forget a use case
- enforcing the default will make less boilerplate

Reply:
But have we got a rule to enforce all use case ?

Reply:
we can use this one instead [@typescript-eslint/switch-exhaustiveness-check](https://typescript-eslint.io/rules/switch-exhaustiveness-check/)
with the following options
```json
{
    allowDefaultCaseForExhaustiveSwitch: false,
    requireDefaultForNonUnion: true,
}
```